docs/CHANGELOG.asciidoc

```diff
@@ -40,10 +41,63 @@ This section summarizes the changes in each release.
 * <<release-notes-7.0.0-alpha1>>
 
 --
+[[release-notes-7.9.1]]
+== {kib} 7.9.1
+
+See also <<breaking-changes-7.9,breaking changes in 7.9>>.
+
+[float]
+[[bug-v7.9.1]]
+=== Bug fixes
+Alerting::
+* Fixes alerts unable to create / update when the name has trailing whitepaces {kibana-pull}76079[#76079]
```

Comment:
I was thinking this would have made the cut for 7.9.1 but it turns out it will be for 7.9.2. This line can be removed and I fixed the labeling in the PR.